packages/@aws-cdk/aws-apigatewayv2-alpha/lib/http/api.ts

```diff
@@ -22,6 +22,20 @@ export interface IHttpApi extends IApi {
    */
   readonly httpApiId: string;
 
+  /**
+   * Default Authorizer to applied to all routes in the gateway
+   * @attribute
+   * @default - No authorizer
+   */
+  readonly defaultAuthorizer?: IHttpRouteAuthorizer;
+
+  /**
+   * Default OIDC scopes attached to all routes in the gateway, unless explicitly configured on the route.
```

Comment:
Please check.

I also added a JWT authorizer to this doc.

https://github.com/aws/aws-cdk/pull/27576/commits/40819a11891185bf4dd08564739f2c923da980ad

Reply:
Can you please provide documentation that describes the usage with a JWT authorizer?
I was only able to verify the [COGNITO_USER_POOLS](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-method.html#cfn-apigateway-method-authorizationscopes) authorizer.

Reply:
Oh... I tried in the integ.lambda, CFn occurred the following error message.

I'll see if I can find any documentation.

```
UPDATE_ROLLBACK_COMPLETE: Resource handler returned message: "Invalid Route authorization type specified. Authorization Scopes are only valid for COGNITO_USER_POOLS and JWT authorization types (Service: AmazonApiGatewayV2; Status Code: 400; Error Code: BadRequestException; Request ID: xxxxxx; Proxy: null)" (RequestToken: xxxxxx, HandlerErrorCode: GeneralServiceException)
```

Reply:
There is `If you specify the AuthorizerId property, specify CUSTOM or COGNITO_USER_POOLS for this property.` in the CFn doc.

I guess this CFn message represents `CUSTOM` as JWT.

What should we do in CDK doc?

Reply:
I found only these articles.

https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html

https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html

Reply:
I'm starting to feel that COGNITO_USER_POOLS is all I need.

Reply:
Yeah, I think that `COGNITO_USER_POOLS` is sufficient since [it generates a `JWT` authorizer](https://github.com/aws/aws-cdk/blob/a67633e102c15741de26237642aed2240bcbc491/packages/%40aws-cdk/aws-apigatewayv2-authorizers-alpha/lib/http/user-pool.ts#L69-L72).

Reply:
In the meantime, please check again as I have only COGNITO_USER_POOLS the message.